build.gradle

```diff
@@ -996,27 +996,23 @@ project('spring-xd-test') {
 		compile "org.springframework.integration:spring-integration-amqp"
 		compile "org.springframework.integration:spring-integration-redis"
 		compile "org.springframework.integration:spring-integration-mqtt"
-		compile("org.springframework.data:spring-data-hadoop") {
-			exclude group: 'org.springframework.batch', module: 'spring-batch-core'
```

Comment:
I think the old excludes were an outdated list - matched the new excludes with what is in the spring-xd-hadoop module minus a few that we need to get the minicluster running for the tests